constraint/go.mod

```diff
@@ -1,32 +1,104 @@
 module github.com/open-policy-agent/frameworks/constraint
 
-go 1.16
+go 1.17
+
+// Prevent otel dependencies from getting out of sync.
+replace (
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.20.0
+	go.opentelemetry.io/otel => go.opentelemetry.io/otel v0.20.0
+	go.opentelemetry.io/otel/metric => go.opentelemetry.io/otel/metric v0.20.0
+	go.opentelemetry.io/otel/sdk => go.opentelemetry.io/otel/sdk v0.20.0
+	go.opentelemetry.io/otel/trace => go.opentelemetry.io/otel/trace v0.20.0
+	go.opentelemetry.io/proto/otlp => go.opentelemetry.io/proto/otlp v0.7.0
+)
 
 require (
-	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d // indirect
 	github.com/davecgh/go-spew v1.1.1
-	github.com/go-openapi/spec v0.20.4 // indirect
 	github.com/golang/glog v1.0.0
 	github.com/google/go-cmp v0.5.6
+	github.com/open-policy-agent/opa v0.37.2
+	github.com/spf13/cobra v1.3.0
+	github.com/spf13/pflag v1.0.5
+	golang.org/x/net v0.0.0-20211209124913-491a49abca63
+	k8s.io/apiextensions-apiserver v0.23.3
+	k8s.io/apimachinery v0.23.3
+	k8s.io/client-go v0.23.3
+	k8s.io/utils v0.0.0-20211116205334-6203023598ed
+	sigs.k8s.io/controller-runtime v0.11.1
```

Comment:
how does this impact the controller-runtime we use in gatekeeper? and will it work with the dynamiccache fork of controller-runtime https://github.com/open-policy-agent/gatekeeper/tree/master/third_party/sigs.k8s.io/controller-runtime?

Reply:
I'll test this locally to see if this upgrade breaks Gatekeeper.

Reply:
I've tested locally, and the upgrade works without impacting Gatekeeper. The dynamiccache fork works correctly.

Reply:
Would we need to upgrade our fork? @shomron

Reply:
Nope - we don't need to. I tested the upgrade and we don't have to modify our fork.